packages/back-end/features/step-definitions/helpers.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-case-declarations */
```

Comment:
That's not very clean but alright (solution would be aving a let variable before and reassign it)